astropy/io/ascii/html.py

```diff
@@ -83,7 +90,10 @@ def process_lines(self, lines):
                                         'installed to read HTML tables')
 
         if 'parser' not in self.html:
-            soup = BeautifulSoup('\n'.join(lines))
+            if lxml_installed:
+                soup = BeautifulSoup('\n'.join(lines),"lxml")
+            else:
+                soup = BeautifulSoup('\n'.join(lines),"html.parser")
```

Comment:
This is a good way of doing it! Just a tiny comment. There should be a space after the comma in the arguments.